website/pages/docs/plugins/destinations/file/overview.mdx

```diff
@@ -10,21 +10,36 @@ import Configuration from "./_configuration.mdx";
 
 <Badge text={"Latest: " + getLatestVersion("destination", "file")}/>
 
-This destination plugin lets you sync data from a CloudQuery source to local files in various formats such as CSV, JSON, Parquet.
+This destination plugin lets you sync data from a CloudQuery source to local files in various formats. It currently supports CSV, line-delimited JSON and Parquet.
 
-This is useful in local environments, but also in production environments where scalability, performance and cost are requirements. For example, this plugin can be used as part of a system that syncs sources across multiple virtual machines, uploads CSV files to a remote storage (such as S3 or GCS), and finally loads them to data lakes such as BigQuery or Athena in batch mode.
+This plugin is useful in local environments, but also in production environments where scalability, performance and cost are requirements. For example, this plugin can be used as part of a system that syncs sources across multiple virtual machines, uploads Parquet files to a remote storage (such as S3 or GCS), and finally loads them to data lakes such as BigQuery or Athena in batch mode. If this is your end goal, you may also want to look at more specific destination cloud storage destination plugins such as [S3](/docs/plugins/destinations/s3/overview), [GCS](/docs/plugins/destinations/gcs/overview) or [Azure Blob Storage](/docs/plugins/destinations/azblob/overview).
 
 ## Example
 
 <Configuration />
 
 ## File Spec
 
-This is the (nested) spec used by the CSV destination Plugin.
+This is the (nested) spec used by the file destination Plugin.
 
-- `directory` (string) (required)
+- `path` (string) (required)
 
-  Directory where all CSV files will be written. A CSV file will be created per table.
+  Path template string that determines where files will be written. The path supports the following placeholder variables:
+
+  - `{{TABLE}}` will be replaced with the table name,
```

Comment:
Maybe get rid of the comma suffixes and the final `and` format for ease of editability/maintenance later? Just leave each bullet point "hanging", without a full stop?

Reply:
Yeah, sounds good